apps/emqx/src/emqx_schema.erl

```diff
@@ -2109,6 +2109,22 @@ common_ssl_opts_schema(Defaults, Type) ->
                     desc => ?DESC(common_ssl_opts_schema_verify)
                 }
             )},
+        {"partial_chain",
+            sc(
+                hoconsc:enum([true, false, two_cacerts_from_cacertfile, cacert_from_cacertfile]),
+                #{
+                    default => Df(partial_chain, false),
+                    desc => ?DESC(common_ssl_opts_schema_partial_chain)
+                }
+            )},
+        {"verify_peer_ext_key_usage",
+            sc(
+                string(),
+                #{
+                    required => false,
+                    desc => ?DESC(common_ssl_opts_verify_peer_ext_key_usage)
+                }
+            )},
```

Comment:
should be added only to listener schema, but not clients.

Reply:
they ({verify_fun, [custom_verify()](https://www.erlang.org/doc/man/ssl#type-custom_verify)}  and {partial_chain, [root_fun()](https://www.erlang.org/doc/man/ssl#type-root_fun)} ) are common in TLS doc, so I think they should be common for both client and server? Their defaults has no impact on the defaults of client.